h/static/scripts/annotator/guest.coffee

```diff
@@ -1,15 +1,7 @@
-$ = require('jquery')
 Annotator = require('annotator')
-Channel = require('jschannel')
+$ = Annotator.$
 
-require('diff-match-patch')
```

Comment:
The guest can function without. The plugins themselves expect these in the
global scope. The entry point module seemed like the only sensible place.
On Mar 26, 2015 9:27 AM, "Nick Stenning" <notifications@github.com> wrote:

> In h/static/scripts/annotator/guest.coffee
> <https://github.com/hypothesis/h/pull/2092#discussion_r27229444>:
>
> >
> > -require('diff-match-patch')
>
> I'm not sure I see the logic of moving these into hypothesis.js. Doesn't
> it make most sense to keep these requirements as close to the code that
> actually uses them as possible?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hypothesis/h/pull/2092/files#r27229444>.
>